pkg/targets/adapter/slacktarget/slack/webapi.go

```diff
@@ -49,16 +49,16 @@ type webAPIClient struct {
 func (c *webAPIClient) Do(methodURL string, data []byte) (Response, error) {
 	method, ok := c.methods[methodURL]
 	if !ok {
-		return nil, fmt.Errorf("Slack method %q not supported", methodURL)
+		return nil, fmt.Errorf("Slack method %q not supported", methodURL) //nolint:stylecheck
```

Comment:
Why did you change the capitalization of `Error` in the above and mark these as `nolint`?  just wondering

Reply:
Because Slack is a brand name so I didn't want to change it to `slack`. That would have made the linter happy too though.